### PR TITLE
Prepare tokio-util 0.5.0 release

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: tokio-util/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 0.5.0 (October 27, 2020)
+
+### Changed
+
+- **Breaking Change**: upgrade to bytes 0.6. `poll_read_buf()` has been removed
+  in favor of a method on tokio's `AsyncReadExt::poll_read_buf()` method.
```
Comment (Darksonn) [inline tokio-util/CHANGELOG.md:6]: Can you split this into several changes? We also don't need the bold "breaking change".

Comment (djc) [reply tokio-util/CHANGELOG.md]: Updated, can you take another look?

File: tokio-util/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# 0.5.0 (October 27, 2020)
+
+### Changed
+
+- Upgrade to bytes 0.6.
+- io: removed `poll_read_buf()` in favor of tokio's `AsyncReadExt::poll_read_buf()`.
+- io: removed `read_buf()` in favor of tokio's `AsyncReadExt::read_buf()`.
```
Comment (Darksonn) [inline tokio-util/CHANGELOG.md:7]:
```suggestion
- Upgrade to bytes 0.6. (#3052)
- io: removed `poll_read_buf()` in favor of Tokio's `AsyncReadExt::poll_read_buf()`. (#3052)
- io: removed `read_buf()` in favor of Tokio's `AsyncReadExt::read_buf()`. (#3052)
```

File: tokio-util/Cargo.toml [hunk 3]
```diff
@@ -44,8 +44,8 @@ pin-project-lite = "0.1.4"
 slab = { version = "0.4.1", optional = true } # Backs `DelayQueue`
 
 [dev-dependencies]
-tokio = { version = "0.3.0", path = "../tokio", features = ["full"] }
-tokio-test = { version = "0.3.0", path = "../tokio-test" }
+tokio = { version = "0.3.0", features = ["full"] }
+tokio-test = { version = "0.3.0" }
```
Comment (Darksonn) [inline tokio-util/Cargo.toml:48]: We keep the `dev-dependencies` as path deps.

Comment (djc) [reply tokio-util/Cargo.toml]: Fixed.

**Comment by carllerche** [comment]
It seems like this is depending on v0.3.2 which isn't released yet. I opened https://github.com/tokio-rs/tokio/pull/3059.

**Comment by carllerche** [comment]
I'm closing this and will open a new PR.

**Pull request closed without merging**